Gems/PhysX/Code/Source/Shape.cpp

```diff
@@ -354,8 +354,12 @@ namespace PhysX
         const physx::PxHitFlags hitFlags = SceneQueryHelpers::GetPxHitFlags(worldSpaceRequest.m_hitFlags);
 
         physx::PxRaycastHit hitInfo;
-        const bool hit = physx::PxGeometryQuery::raycast(start, unitDir, m_pxShape->getGeometry().any(), pose,
-                                                         worldSpaceRequest.m_distance, hitFlags, maxHits, &hitInfo);
+        bool hit;
```

Comment:
I bet hultonha would suggest an immediately invoked lambda here so you can make hit const.

Reply:
hahaha, I want to keep this one simple because I have to resolve this file now in the PhysX5Support branch.

Reply:
😂 I would have said that, there's also the `AZStd::invoke` pattern which can be used to make the callsite clearer if you prefer, but this is okay for now (might want to initialize `bool hit` to a known value first though)